### ISSUE TITLE
Learning rate finetune LoRA?

### ISSUE THREAD

**Issue by huynhbaobk** [QwenLM/Qwen2-VL#127]
I found the different `learning_rate` in your README when finetune LoRA is **5e-6** , but in LLama Factory - _qwen2vl_lora_sft.yaml_ is **1e-4**. Could you clarify which learning rate was actually used in your experiment?

**Comment by cjfcsjt** [comment]
@ShuaiBai623 Could someone please clarify this? Thanks.

**Comment by syspider** [comment]
mark

**Comment by hiyouga** [comment]
Fine-tuning LoRA on few examples (<5000) should use larger LR, so we adopt 1e-4 as the default LR in llamafactory
